### PR TITLE
I have created the 'validatePassword' function

### PR DESCRIPTION
**Pull request by Artanis7488**
In this code, I have created a password validation function and written tests to verify its functionality.

In "constant.tsx," I have defined a constant: "passwordCompare" (an error message in case of non-matching passwords).

In "Functions.tsx," I have created the "validatePasswords" function that compares two passwords and returns an error message or "null" if they match.

In "function.test.tsx," I have used Jest to write two tests: the first one checks that the function returns "null" if the passwords match, and the second one checks that it returns the error message if they don't match.

### CODE REVIEW

File: frontend/src/utilities/Functions.tsx [hunk 1]
```diff
@@ -1 +1,10 @@
-//TODO: Create function reutilization file for all the constant values
\ No newline at end of file
+//TODO: Create function reutilization file for all the constant values
+import { passwordCompare } from "./Constant";
+
+export function validarContraseñas(contraseña1: string, contraseña2: string): string | null {
+    if (contraseña1 === contraseña2) {
+      return null; // Las contraseñas coinciden
```
Comment (ekimike) [inline frontend/src/utilities/Functions.tsx:6]: Why are you returning a null value when the comparison was successful on a method which its name suggests to perform a validation, which should result a boolean value, TRUE when success and FALSE when wrong.

File: frontend/src/utilities/Functions.tsx [hunk 2]
```diff
@@ -1 +1,10 @@
...
+    } else {
+      return passwordCompare;
```
Comment (ekimike) [inline frontend/src/utilities/Functions.tsx:8]: Image being a new developer on this team who reads this piece of code, what do you think him/her will infer based on the name `passwordCompare`?  It seems more to a method name than a result.

**Review by ekimike — CHANGES_REQUESTED** [review]

**Review by AnthonyRicse98 — APPROVED** [review]

**Review by Anthonyricse — APPROVED** [review]

File: frontend/src/utilities/Functions.tsx [hunk 3]
```diff
@@ -1 +1,10 @@
-//TODO: Create function reutilization file for all the constant values
\ No newline at end of file
+//TODO: Create function reutilization file for all the constant values
+import { passwordCompare } from "./Constant";
+
+export function validarContraseñas(contraseña1: string, contraseña2: string): string | null {
```
[the remark this answers is not in the document]
Comment (Artanis7488) [reply frontend/src/utilities/Functions.tsx]: I already switched it to the English version.

File: frontend/src/utilities/Functions.tsx (hunk 1 above)
Comment (Artanis7488) [reply frontend/src/utilities/Functions.tsx]: You're right, now it returns true or false.

File: frontend/src/utilities/Constant.tsx [hunk 4]
```diff
@@ -1,3 +1,4 @@
 export const nameRegex = /^[a-zA-ZÀ-ÿ\s'-]{1,150}$/;
 export const emailRegex = /^[\w-\.]+@([\w-]+\.)+[\w-]{2,4}$/;
-export const passwordRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
\ No newline at end of file
+export const passwordRegex = /^(?=.*[a-z])(?=.*[A-Z])(?=.*\d)(?=.*[@$!%*?&])[A-Za-z\d@$!%*?&]{8,}$/;
+export const passwordCompare = 'Las contraseñas no coinciden';
```
[the remark this answers is not in the document]
Comment (Artanis7488) [reply frontend/src/utilities/Constant.tsx]: Done, professor. I've already changed the name to 'passwordMismatchError'

File: frontend/src/test/function.test.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  it('Debería devolver nulo si las contraseñas coinciden', () => {
+    const contraseña1 = 'contraseña123';
+    const contraseña2 = 'contraseña123';
+    const resultado = validarContraseñas(contraseña1, contraseña2);
+    expect(resultado).toBeNull();
+  });
+
+  it('Debería devolver un mensaje de error si las contraseñas no coinciden', () => {
+    const contraseña1 = 'contraseña123';
+    const contraseña2 = 'otracontraseña';
```
[the remark this answers is not in the document]
Comment (Artanis7488) [reply frontend/src/test/function.test.tsx]: I already changed it into English, professor.

File: frontend/src/test/function.test.tsx [hunk 6]
```diff
@@ -0,0 +1,18 @@
+import { validarContraseñas } from '../utilities/Functions';
+import { passwordCompare } from '../utilities/Constant';
+
+describe('Validar Contraseñas', () => {
+  it('Debería devolver nulo si las contraseñas coinciden', () => {
+    const contraseña1 = 'contraseña123';
+    const contraseña2 = 'contraseña123';
```
[the remark this answers is not in the document]
Comment (Artanis7488) [reply frontend/src/test/function.test.tsx]: I already changed it into English, professor.

**Review by ekimike — APPROVED** [review]

**Pull request merged**